app/javascript/actionsPanel/actionsPanel.js

```diff
@@ -1,3 +1,4 @@
+import { toggleFlagUserModal } from '../packs/flagUserModal';
 import { request } from '@utilities/http';
```

Comment:
code climate is complaining about this order of imports but when I change the order and commit the file eslint in husky hook changes this order back.